app.js

```diff
@@ -22,4 +22,31 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', async (req, res) => {
+  try {
```

Comment:
nice use of the try catch, if its better for you, you can working with it the next days :)